src/Discord.Net.Core/Entities/Messages/SticketFormatType.cs

```diff
@@ -0,0 +1,15 @@
+namespace Discord
+{
+    /// <summary> Defines the types of formats for stickers. </summary>
+    public enum StickerFormatType
+    {
+        /// <summary> Default value for a sticker format type. </summary>
+        None = 0,
+        /// <summary> The sticker format type is png. </summary>
+        Png = 1,
+        /// <summary> The sticker format type is apng. </summary>
+        Apng = 2,
+        /// <summary> The sticker format type is lottie. </summary>
+        Lottie = 3,
```

Comment:
Could maybe add what a "lottie" is

Reply:
Wouldn't it be safe to assume they will know these are image formats? And imo searching about it would probably give better information than describing it here because I'm not sure saying something around "lottie is an open JSON-based vector animation format" would help much, unless you had something else in mind. 🤔